CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 0.21.0
+- Allow use with future unknown Rails versions
+- Add support for Rails 8.0
+
```

Comment:
Would need to mention that Rails 7.0 support is also dropped (or untested)

Reply:
I'll make the necessary changes to the changelog after we decide on the way forward